todo-backend/README.adoc

```diff
@@ -339,28 +425,48 @@ jboss-eap/eap-xp3       ...             ...             A Helm chart to build an
 ----
 endif::[]
 
+
+ifdef::ProductRelease,EAPXPRelease[]
 === Deploy a PostgreSQL Database on OpenShift
 
 [source,options="nowrap"]
 ----
 $ oc new-app postgresql-ephemeral \
-   -p DATABASE_SERVICE_NAME=todos-db \
+   -p DATABASE_SERVICE_NAME=todo-backend-db \
    -p POSTGRESQL_DATABASE=todos
 ----
 
-This will create a PostgreSQL database named `todos` on OpenShift that can be accessed on the port `5432` on the service `todos-db`.
-We don't need to copy the credentials to connect to the database as we will retrieve them later using the `todos-db` secret that was created
+This will create a PostgreSQL database named `todos` on OpenShift that can be accessed on the port `5432` on the service `todo-backend-db`.
+We don't need to copy the credentials to connect to the database as we will retrieve them later using the `todo-backend-db` secret that was created
 when the database is deployed.
+endif::[]
 
-ifndef::ProductRelease,EAPXPRelease[]
+ifndef::ProductRelease[]
 === Build and Deploy the Backend on OpenShift with Bootable Jar
 
+==== Deploy a PostgreSQL Database on OpenShift
+
+[source,options="nowrap"]
+----
+$ oc new-app postgresql-ephemeral \
+   -p DATABASE_SERVICE_NAME=todo-backend-db \
+   -p POSTGRESQL_DATABASE=todos
+----
+
+This will create a PostgreSQL database named `todos` on OpenShift that can be accessed on the port `5432` on the service `todo-backend-db`.
+We don't need to copy the credentials to connect to the database as we will retrieve them later using the `todo-backend-db` secret that was created
+when the database is deployed.
+
+==== Build and Deploy the Backend on OpenShift with Bootable Jar using the Helm Chart
+
 The backend will be built and deployed on OpenShift with a Helm Chart for {productName}.
 
+ifndef::ProductRelease,EAPXPRelease[]
+
 [source,options="nowrap",subs="+attributes"]
 ----
 $ helm install todo-backend --set build.ref={WildFlyQuickStartRepoTag} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
```

Comment:
The `-set build.ref={WildFlyQuickStartRepoTag}` can be omitted in case the https://github.com/wildfly/wildfly-charts/pull/103 PR gets merged.